DOCS/interface-changes.rst

```diff
@@ -27,6 +27,7 @@ Interface changes
       ("") uses the value of --hwdec.
     - drop deprecated --ad/--vd features
     - drop deprecated --sub-codepage syntax
+    - add --ontop-level option for macOS
```

Comment:
New features are normally not announced in this file (unless they're very central), but it's not wrong to do so either. The commit I just pushed makes this probably conflict, so feel free to just drop it.